sql/moz-fx-data-shared-prod/telemetry/mobile_usage_2021/view.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.mobile_usage_2021`
```

Comment:
This table is named 2021, but it has data since 2017. Should be be called `mobile_usage_yearly` instead?

Reply:
This is a good thing to discuss. I likely should add some more prose to the description about this.

This table is primarily aimed at serving source of truth for KPI data in 2021. For 2019 and 2020, we used `firefox_nondesktop_exact_mau28` for that purpose, but the schema there has become a bit stale. I want to express that this table is source of truth for reporting in 2021, but that KPI reporting needs and measures may change in future years, necessitating a change in schema. So I want this user-facing view to be obviously 2021-specific.

There's also the Shredder problem here. The data that we backfill will not reflect users who have sent deletion requests, whereas the incremental update through the year will not reflect deletion requests. This view should give stable results; I don't want numbers to drop due to new backfills. Although for the mobile case, I expect we may need to do at least one backfill in coming months as the underlying Glean ETL matures.